subsystems/sidecar/index.js

```diff
@@ -56,14 +55,6 @@ const ops = {
   Touch: require('./ops/touch')
 }
 
-class PermitStore extends Store {
```

Comment:
Lets remove subsystems/sidecar/lib/store too. Also replace the store in sidecar/state